### ISSUE TITLE
Version parsing fails on GKE

### ISSUE THREAD

**Issue by gytisgreitai** [ksonnet/ksonnet#236]
When running `ks -v apply --gc-tag "ksnt" test` I get an error:

`ERROR strconv.Atoi: parsing "8+": invalid syntax`

This seems to stem from server version parsing: https://github.com/ksonnet/ksonnet/blob/b70c996f6c2af98ddf6f61e0a25cc0e7e01114e6/utils/meta.go#L27

And there is even a test which ensures that 6x is not parsed:

https://github.com/ksonnet/ksonnet/blob/b70c996f6c2af98ddf6f61e0a25cc0e7e01114e6/utils/meta_test.go#L29

The problem is that on GKE minor is returned as 8+:

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"8", GitVersion:"v1.8.3", GitCommit:"f0efb3cb883751c5ffdbe6d515f3cb4fbe7b7acd", GitTreeState:"clean", BuildDate:"2017-11-08T18:39:33Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"8+", GitVersion:"v1.8.4-gke.0", GitCommit:"04502ae78d522a3d410de3710e1550cfb16dad4a", GitTreeState:"clean", BuildDate:"2017-11-27T19:19:56Z", GoVersion:"go1.8.3b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by gytisgreitai** [comment]
Something like this seems to help, not sure though why the strict version parsing:

```
ret.Minor, err = strconv.Atoi(numbersOnly(v.Minor))
< ... snip ... >
func numbersOnly(value string) string {
	return strings.Map(func(r rune) rune {
		if r < '0' || '9' < r {
			return -1
		}
		return r
	}, value)
}
```

**Comment by kyamazawa** [comment]
+1

**Comment by kyamazawa** [comment]
[PR](https://github.com/ksonnet/ksonnet/pull/254) created.

**Comment by jessicayuen** [comment]
Fixed in #254
